webapp/packages/core-ui/src/Tabs/TabsBox/TabsBox.tsx

```diff
@@ -6,75 +6,59 @@
  * you may not use this file except in compliance with the License.
  */
 import { forwardRef, PropsWithChildren, ReactNode } from 'react';
-import styled, { css } from 'reshadow';
 
-import { useStyles } from '@cloudbeaver/core-blocks';
-import type { ComponentStyle } from '@cloudbeaver/core-theming';
+import { s, SContext, StyleRegistry, useS } from '@cloudbeaver/core-blocks';
 import type { MetadataMap } from '@cloudbeaver/core-utils';
 
+import { tabPanelStyles } from '../..';
 import type { ITabData } from '../TabsContainer/ITabsContainer';
 import { TabsState } from '../TabsState';
+import styles from './TabsBox.m.css';
 
-const styles = css`
-  tabs-box {
-    display: flex;
-    flex-direction: column;
-    flex: 1;
-    overflow: hidden;
-  }
-  tabs {
-    display: flex;
-    flex-direction: row;
-    flex: 0 0 auto;
-    overflow: auto;
-  }
-  tab-panels {
-    flex: 1;
-    display: flex;
-    overflow: hidden;
-  }
-  tab-panel {
-    flex: 1;
-    display: flex;
-    overflow: hidden;
-    outline: none;
-    position: relative;
-  }
-`;
+const tabsBoxRegistry: StyleRegistry = [
+  [
+    tabPanelStyles,
+    {
+      mode: 'append',
+      styles: [styles],
+    },
+  ],
+];
 
 type TabsBoxProps = PropsWithChildren<{
   currentTabId: string | null;
   tabs?: ReactNode;
   tabIndex?: number;
   localState?: MetadataMap<string, any>;
   tabList?: string[];
+  tabsClassName?: string;
   enabledBaseActions?: boolean;
   autoSelect?: boolean;
   className?: string;
   onChange?: (tab: ITabData<any>) => void;
-  style?: ComponentStyle;
 }>;
 
 export const TabsBox = forwardRef<HTMLDivElement, TabsBoxProps>(function TabsBox(
```

Comment:
this component must be deprecated or removed if possible

Reply:
marked it as deprecated cause this is a mounting of changes appeared, and moreover this is out of the ticket scope to remove it

Reply:
thank you